govcd/vdc.go

```diff
@@ -555,99 +555,6 @@ func (vdc *Vdc) CreateRawVApp(name string, description string) (*VApp, error) {
 	return vapp, nil
 }
 
-// ComposeVApp creates a vapp with the given template, name, and description
-// that uses the storageprofile and networks given. If you want all eulas
-// to be accepted set acceptalleulas to true. Returns a successful task
-// if completed successfully, otherwise returns an error and an empty task.
-// Deprecated: bad implementation
-func (vdc *Vdc) ComposeVApp(orgvdcnetworks []*types.OrgVDCNetwork, vapptemplate VAppTemplate, storageprofileref types.Reference, name string, description string, acceptalleulas bool) (Task, error) {
```

Comment:
We don't  usually remove things due to semver. It is deprecated, and we can be not using it, but semantinc versioning usually implies that a major version bump is required to remove things.

Reply:
Put it back in place. Please approve so I can merge 😄